packages/cli/internal/pkg/cli/logs_engine.go

```diff
@@ -45,10 +72,29 @@ func (o *logsEngineOpts) Execute() error {
 	}
 
 	logGroupName := contextInfo.EngineLogGroupName
-	if o.tail {
-		err = o.followLogGroup(logGroupName)
+	log.Debug().Msgf("Engine log group name: '%s'", logGroupName)
+
+	if o.workflowRunId != "" {
+		log.Debug().Msgf("Getting log stream for workflow run '%s'", o.workflowRunId)
+
+		workflowRunLog, err := o.workflowManager.GetEngineLogByRunId(o.workflowRunId)
```

Comment:
Some engines can return an empty list of coudwatch streams. In this case the downstream code will dump complete engine logs. We should handle this edge case.